src/core/ad.jl

```diff
@@ -153,31 +140,7 @@ function verifygrad(grad::AbstractVector{<:Real})
     end
 end
 
-for F in (:link, :invlink)
-    @eval begin
-        function $F(
-            dist::Dirichlet, 
-            x::Tracker.TrackedArray, 
-            ::Type{Val{proj}} = Val{true}
-        ) where {proj}
-            return Tracker.track($F, dist, x, Val{proj})
-        end
-        Tracker.@grad function $F(
-            dist::Dirichlet, 
-            x::Tracker.TrackedArray, 
-            ::Type{Val{proj}}
-        ) where {proj}
-            x_data = Tracker.data(x)
-            T = eltype(x_data)
-            y = $F(dist, x_data, Val{proj})
-            return  y, Δ -> begin
-                out = (ForwardDiff.jacobian(x -> $F(dist, x, Val{proj}), x_data)::Matrix{T})' * Δ
-                return (nothing, out, nothing)
-            end
-        end
-    end
-end
-
+# These still seem necessary
```

Comment:
Should this maybe be moved into Bijectors.jl? (Though prob in a different PR)

Reply:
Actually Bijectors differentiates fine without these. There seems to be something at the Turing-Bijectors interface that's causing issues with Tracker. I didn't have time to dig deeper into the issue so I just brought back those for now.